### PR TITLE
Add list of community SDKs to homepage.

### PR DESCRIPTION
**Pull request by 10thfloor**
Completing:  #471

## Description

Adds a link to the community sdks page to the homepage.

### CODE REVIEW

File: docs/content/index.mdx [hunk 1]
```diff
@@ -23,6 +23,10 @@ Follow the <TrackingLink href="/flow-js-sdk/flow-app-quickstart/" eventName="Hom
 The Kitty Items application is a full-featured NFT marketplace that demonstrates what a full-stack application can look like on Flow. It's designed to showcase the power of Cadence and Flow's client libraries.
 Have a look at the [Kitty Items GitHub repository](https://github.com/onflow/kitty-items) for more details.
 
+### 🏋️‍♀️ Community SDKs
+
+Check out the [a list of community SDKs](/sdks) currently in development, Including SDKs for [Ruby](/sdks#ruby) [.NET](/sdks#net), [JVM](/sdks#jvm), and others!
```
Comment (10thfloor) [inline docs/content/index.mdx:28]:
```suggestion
Check out the [list of community SDKs](/sdks) currently in development, Including SDKs for [Ruby](/sdks#ruby) [.NET](/sdks#net), [JVM](/sdks#jvm), and others!
```

File: docs/content/sdks.mdx [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+THe following is a list of SDK;s from the FLow community that are currently in development.
+
+Legend:
+
+- 😀: Activity in the last six months
+- 📘: Documentation
+
+Want your SDK listed here? Please fork the [`flow` repository](https://github.com/onflow/flow), edit [this page](https://github.com/onflow/flow/blob/master/docs/content/sdks.md), and submit a pull request.
+
+## JVM (Java, Kotlin, Scala, etc.)
```
Comment (turbolent) [inline docs/content/sdks.mdx:14]: Why were the Go and JS SDKs removed, but the JVM one kept? If it's only community ones, maybe remove it too?

File: docs/content/sdks.mdx [hunk 3]
```diff
@@ -0,0 +1,54 @@
+# SDKs
```
Comment (turbolent) [inline docs/content/sdks.mdx:1]: If this pages is now only for community SDKs, maybe adjust the header here

File: docs/content/status.mdx [hunk 4]
```diff
@@ -18,20 +18,20 @@ import {
 <NetworkStatus
```
Comment (turbolent) [inline docs/content/status.mdx:18]: Maybe submit these changes in a different PR, they are unrelated

File: docs/content/status.mdx [hunk 5]
```diff
@@ -18,20 +18,20 @@ import {
...
   networkVersion="9"
-  versionUrl=""
+  versionUrl="https://docs.onflow.org/node-operation/past-sporks/#mainnet-9"
   nextSporkDate="June 23, 2021"
 />
 <NetworkStatus
   networkName={TESTNET}
-  networkVersion="9"
-  versionUrl=""
+  networkVersion="Devne23"
```
Comment (turbolent) [inline docs/content/status.mdx:26]: Why is the network called "Devne" and not "Devnet"?

Looks like there was a typo in 22 and then it was just copied?
https://docs.onflow.org/node-operation/past-sporks/#devne22

File: docs/content/sdks.mdx (hunk 2 above)
Comment (10thfloor) [reply docs/content/sdks.mdx:14]: JVM in in house?

File: docs/content/status.mdx (hunk 5 above)
Comment (10thfloor) [reply docs/content/status.mdx:26]: Not sure how that got in there. 
This is what Vishal has written.

File: docs/content/sdks.mdx (hunk 3 above)
Comment (10thfloor) [inline docs/content/sdks.mdx:1]:
```suggestion
# Community SDKs
```

File: docs/content/status.mdx (hunk 5 above)
Comment (10thfloor) [reply docs/content/status.mdx:26]: Yup looks like it. I'll make another PR to fix this.

**Comment by turbolent** [comment]
Thanks Mack!

File: docs/content/sdks.mdx (hunk 2 above)
Comment (turbolent) [reply docs/content/sdks.mdx:14]: LOL, no idea what to make of this

File: docs/content/status.mdx (hunk 5 above)
Comment (turbolent) [reply docs/content/status.mdx:26]: Thanks Mack!
